azurerm/internal/services/containers/container_group_resource.go

```diff
@@ -1372,3 +1411,63 @@ func resourceArmContainerGroupPortsHash(v interface{}) int {
 
 	return hashcode.String(buf.String())
 }
+
+func flattenContainerGroupDnsConfig(input *containerinstance.DNSConfiguration) []interface{} {
+	output := make(map[string]interface{})
+	var outputArr []interface{}
```

Comment:
Can we remove this local variable here so as to be consistent with other flatten functions? ([example](https://github.com/terraform-providers/terraform-provider-azurerm/blob/aad52564c4fad1197ed4ab03f62e35d0ffc0f63b/azurerm/internal/services/containers/container_group_resource.go#L979))